### PR TITLE
chore: rename wallet-js-client to wallet-adapter-web

### PR DESCRIPTION
**Pull request by sudeshrshetty**
Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: cmd/rp-adapter-vue/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.14.0",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/vue-fontawesome": "^0.1.10",
-    "@trustbloc/wallet-js-client": "0.1.6",
+    "@trustbloc-cicd/wallet-adapter-web": "0.1.7-snapshot-0bd7297",
```
Comment (fqutishat) [inline cmd/rp-adapter-vue/package.json:16]: why just reference wallet-adapter-web directly

Comment (fqutishat) [reply cmd/rp-adapter-vue/package.json:16]: it's in same project

**Review by troyronda — APPROVED** [review]

File: cmd/rp-adapter-vue/package.json (hunk 1 above)
Comment (fqutishat) [reply cmd/rp-adapter-vue/package.json:16]: no need for snapshot

Comment (fqutishat) [reply cmd/rp-adapter-vue/package.json:16]: use ../

Comment (sudeshrshetty) [reply cmd/rp-adapter-vue/package.json:16]: I noticed when we do npm install local it installs all contents outside webpack dist, so it is gets heavy.

**Review by fqutishat — APPROVED** [review]

**Pull request merged**